Iterable-iOS-SDK/IterableAPI.h

```diff
@@ -270,14 +270,28 @@ typedef NS_ENUM(NSInteger, PushServicePlatform) {
  
  @param dataFields              Data fields to store in the user profile
  @param mergeNestedObjects      Merge top level objects instead of overwriting
- @param onSuccess               OnSuccessHandler to invoke if disabling the token is successful
- @param onFailure               OnFailureHandler to invoke if disabling the token fails
+ @param onSuccess               OnSuccessHandler to invoke if update is successful
+ @param onFailure               OnFailureHandler to invoke if update fails
  
  @see OnSuccessHandler
  @see OnFailureHandler
  */
 - (void)updateUser:(NSDictionary *)dataFields mergeNestedObjects:(BOOL)mergeNestedObjects onSuccess:(OnSuccessHandler)onSuccess onFailure:(OnFailureHandler)onFailure;
 
+/*!
+ @method
+
+ @abstract Updates the current user's email.
```

Comment:
also maybe add a comment to the effect that it'll update the current email in this IterableApi instance